shell/components/ExplorerProjectsNamespaces.vue

```diff
@@ -284,6 +287,21 @@ export default {
 
       return location;
     },
+
+    createNamespaceLocationFlatList() {
+      const location = this.createNamespaceLocationOverride ? { ...this.createNamespaceLocationOverride } : {
+        name:   'c-cluster-product-resource-create',
+        params: {
+          product:  this.$store.getters['currentProduct'].name,

```

Comment:
Are we sure to always have a product and a name? We should add always optional chaining as the object may not exists.